README.md

```diff
@@ -109,7 +109,7 @@ kubernetes cluster (via `kubectl`). When executing tests from IDE make sure that
 
 ```bash
 minikube start
-./mvnw clean verify -Pe2e
+./mvnw clean verify -Pcassandra,elasticsearch,all-in-one
```

Comment:
> ES tests are passing when cassandra tests are disabled.

Should one expect this command to finish successfully, or will this fail as per your comment on the PR ?

Reply:
locally it passes fine, the problem is with travis.

This is a one liner to run everything